docs/API.md

```diff
@@ -0,0 +1,107 @@
+Generally when using the API npm installation or try...catch blocks are encouraged to not break your app in case RUM library isn't loaded for some reason.
```

Comment:
Not sure I understand this sentence. Who is "encouraged"?

Reply:
Application developers who integrate splunk rum to their application are encouraged to use those methods. Since API documentation is mostly visited by developers anyway it's pretty much implied to be the target audience